### PR TITLE
Create Maven Parent Module

### PR DESCRIPTION
**Pull request by eolivelli**
Prepare a modular Maven project structure

### CODE REVIEW

**Comment by eolivelli** [comment]
This is a first step.
The next step will be to introduce a new module with a standalone runner.
Then we can update the 'getting started' guide

File: web/src/main/webapp/META-INF/context.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Context path="/bookkeeper-visual-manager"/>
```
Comment (mino181295) [inline web/src/main/webapp/META-INF/context.xml]: This is "/"

Comment (eolivelli) [reply web/src/main/webapp/META-INF/context.xml]: done

**Comment by eolivelli** [comment]
@mino181295 I have addressed your comment PTAL

**Pull request merged**